PhysicsTools/PatAlgos/plugins/PATElectronSlimmer.cc

```diff
@@ -136,6 +145,23 @@ pat::PATElectronSlimmer::produce(edm::Event & iEvent, const edm::EventSetup & iS
         if (dropSaturation_(electron)) { electron.setSaturationInfo(reco::GsfElectron::SaturationInfo()); }
         if (dropExtrapolations_(electron)) { electron.setTrackExtrapolations(reco::GsfElectron::TrackExtrapolations());  }
         if (dropClassifications_(electron)) { electron.setClassificationVariables(reco::GsfElectron::ClassificationVariables()); electron.setClassification(reco::GsfElectron::Classification()); }
+
+        if(computeMiniIso_){
+            pat::PFIsolation miniiso;
+            if(fabs(electron.isEE()))
+                miniiso = pat::getMiniPFIsolation(pc.product(), electron.p4(),
+                                                  miniIsoParamsE[0], miniIsoParamsE[1], miniIsoParamsE[2],
+                                                  miniIsoParamsE[3], miniIsoParamsE[4], miniIsoParamsE[5],
+                                                  miniIsoParamsE[6], miniIsoParamsE[7], miniIsoParamsE[8]);
+            else
+                miniiso = pat::getMiniPFIsolation(pc.product(), electron.p4(),
+                                                  miniIsoParamsB[0], miniIsoParamsB[1], miniIsoParamsB[2],
+                                                  miniIsoParamsB[3], miniIsoParamsB[4], miniIsoParamsB[5],
+                                                  miniIsoParamsB[6], miniIsoParamsB[7], miniIsoParamsB[8]);
+            electron.setMiniPFIsolation(miniiso);
```

Comment:
For the comment about being in the slimmers, I mainly put them there because the mini-iso algo uses information stored in the packed PF candidates (mainly, the fromPV() function). If I can get equivalent information from the reco::PFcandidates, then I think the computation could be moved to the producers.